raiden/tests/conftest.py

```diff
@@ -23,7 +23,7 @@
 def pytest_addoption(parser):
     parser.addoption(
         '--blockchain-type',
-        choices=['geth'],
+        choices=['geth', 'parity'],
```

Comment:
This list/tuple is used in other places too. Like [here](https://github.com/raiden-network/raiden/pull/3483/files#diff-8e4c09e9a7222a70cd9463af4c512a23R51). Shouldn't we put it in the settings.py as `SUPPORTED_CLIENTS` so that it's hard-coded in a common place?